### PR TITLE
test: squad page

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- A large test coverage for the Squad feed page.
- The "Create new post" modal should have its own test suite due to its complexity, we already have tickets for missing test suites (WT-1064).
- The options menu is acting weird (throwing error on test), we will investigate further on how to test it when we get more time but this test suite should allow us to feel a bit safer when making changes.
- Fixed missing tooltips on clickable icons.
- Refactored unnecessary wrapping elements.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1067 #done

### CODE REVIEW

File: packages/webapp/__tests__/SquadFeedPage.tsx [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+//   it('should have a feedback button on mobile devices', async () => {
+//     renderComponent();
+//     const options = await screen.findByLabelText('Squad options');
+//     options.click();
+//     const feedback = await screen.findByLabelText('squad-options-feedback');
+//     expect(feedback).toHaveClass('flex tablet:hidden');
+//     expect(feedback).toHaveAttribute('href', link);
+//   });
+// });
```
Comment (rebelchris) [inline packages/webapp/__tests__/SquadFeedPage.tsx:380]: This will block for now.
Either we can remove it and log as separate or hold this PR.

**Review by rebelchris — CHANGES_REQUESTED** [review]
Just one comment, otherwise looks great, thanks for the massive efforts on this.

File: packages/webapp/__tests__/SquadFeedPage.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/webapp/__tests__/SquadFeedPage.tsx:380]: I agree. Alright, let's remove this for now as the investigation around context menu not showing can impact many areas. Thanks for the note.

Comment (sshanzel) [reply packages/webapp/__tests__/SquadFeedPage.tsx:380]: Logged! ✅

**Comment by sshanzel** [comment]
@rebelchris I have now updated the unused tests. Would love to hear what you think! 🚢

**Review by rebelchris — APPROVED** [review]
Nice one Lee! 🤘

**Pull request merged**